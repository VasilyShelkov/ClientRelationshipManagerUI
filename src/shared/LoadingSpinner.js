import React from 'react';
import CircularProgress from 'material-ui/CircularProgress';

export default () => (
  <div>
    <CircularProgress size={80} thickness={5} />
  </div>
);
