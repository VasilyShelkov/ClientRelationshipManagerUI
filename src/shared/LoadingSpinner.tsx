import * as React from 'react';
import { CircularProgress } from 'material-ui';

export default () => (
  <div>
    <CircularProgress size={80} thickness={5} />
  </div>
);
