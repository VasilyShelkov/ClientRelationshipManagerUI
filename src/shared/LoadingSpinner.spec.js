import React from 'react';
import renderer from 'react-test-renderer';
import LoadingSpinner from './LoadingSpinner';

describe('src/shared/LoadingSpinner.js', () => {
  it('renders the loading spinner', () => {
    expect(renderer.create(<LoadingSpinner />).toJSON())
      .toMatchSnapshot();
  });
});
