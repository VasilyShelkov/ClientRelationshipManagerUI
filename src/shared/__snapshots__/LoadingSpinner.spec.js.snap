exports[`src/shared/LoadingSpinner.js renders the loading spinner 1`] = `
<div>
  LOADING
</div>
`;
